pxtextension.ts

```diff
@@ -75,7 +71,20 @@ namespace mlrunner {
       }
     }
   }
-  // End simulator code.
+}
+// End simulator code.
+
+
+//% fixedInstances
+//% blockNamespace=mlrunner
+class MlEvent {
```

Comment:
Not sure why, but it didn't work if the class was defined inside the namespace, so had to break it out.